api/api_public_blockchain.go

```diff
@@ -504,9 +504,21 @@ type StructLogRes struct {
 }
 
 // formatLogs formats EVM returned structured logs for json output
-func FormatLogs(logs []vm.StructLog) []StructLogRes {
+func FormatLogs(timeout time.Duration, logs []vm.StructLog) ([]StructLogRes, error) {
+	logTimeout := false
+	deadlineCtx, cancel := context.WithTimeout(context.Background(), timeout)
+	go func() {
+		<-deadlineCtx.Done()
+		logger.Error("[Trace logger] Timeout reached", "timeout", timeout, "err", deadlineCtx.Err())
```

Comment:
```suggestion
		logger.Debug("trace logger timeout", "timeout", timeout, "err", deadlineCtx.Err())
```
Error level is too verbose. For reference, failed RPC calls are logged at Debug level (https://github.com/klaytn/klaytn/pull/1735).